pom.xml

```diff
@@ -615,7 +615,8 @@ flexible messaging model and an intuitive client API.</description>
             <exclude>**/*.crt</exclude>
             <exclude>**/*.key</exclude>
             <exclude>**/*.csr</exclude>
-            <exclude>**/proto/*.java</exclude>
+            <exclude>src/main/java/org/apache/bookkeeper/mledger/proto/MLDataFormats.java</exclude>
```

Comment:
formatting?

Reply:
I checked. These are generated and a license is not generated. This is ok.